.github/workflows/pr-labeler.yml

```diff
@@ -1,13 +1,38 @@
 name: Apply labels to PR
 on:
   pull_request_target:
-    types: [opened]
+    # required for pr-labeler to support PRs from forks
+    # ===================== ⛔ ☢️ 🚫 ⚠️ Warning ⚠️ 🚫 ☢️ ⛔ =======================
+    # Be very careful what you put in this GitHub Action workflow file to avoid
+    # malicious PRs from getting access to the Mermaid-js repo.
+    #
+    # Please read the following first before reviewing/merging:
+    # - https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#pull_request_target
+    # - https://securitylab.github.com/research/github-actions-preventing-pwn-requests/
+    types: [opened, reopened, synchronize]
+
+  pull_request:
+    # Only needed for testing: the `pull_request_target` action will also run
+    paths:
+      - .github/workflows/pr-labeler.yml
+      - .github/pr-labeler.yml
+    types: [opened, reopened, synchronize]
```

Comment:
Should we remove this before merge?

Reply:
Yep!

Normally, I'd say keep it, so any future PRs that modify these files would be tested, but it turns out the `release-drafter/release-drafter` action only uses the config file in the `develop` branch anyway, so there's no point in keeping this.

I've dropped commit https://github.com/mermaid-js/mermaid/commit/f6ee5215b2800f3e4b2e81d19eb7b863cc3225b3 and force pushed to  https://github.com/mermaid-js/mermaid/pull/4868/commits/99beeba261674f57249179df6ad415e8a7169e07